doc/lua_api.txt

```diff
@@ -2092,15 +2092,17 @@ Elements
 
 * Show an inventory image of registered item/node
 
-### `bgcolor[<color>]`
-
-* Sets background color of formspec as `ColorString`
-
-### `bgcolor[<color>;<fullscreen>]`
-
-* If `color` is a valid `ColorString`, the fullscreen background color
-  is set to `color`.
-* If `fullscreen` is a true value, the fullscreen background color is drawn.
+### `bgcolor[<bgcolor>;<fullscreen>;fbgcolor]`
+
+* Sets background color of formspec.
+* `bgcolor` and `fbgcolor` are `ColorString`s, they define the color of the
+  non-fullscreen and the fullscreen background.
+* `fullscreen` can be one of the following:
+  * `false`: Only the non-fullscreen background-color is drawn. (default)
+  * `true`: Only the fullscreen background-color is drawn.
+  * `both`: The non-fullscreen and the fullscreen background-color are drawn.
+  * `neither`: No background-color is drawn.
```

Comment:
The dash is not needed for `background color`. It's unusual to read it this way, but can't say whether it's "proper" English.

Reply:
Done.